src/client/helpers/helpers.js

```diff
@@ -287,6 +296,21 @@ function deletePendingMemories(cb) {
     cb();
```

Comment:
Every time something is deleted I think! Haha - I think we discussed before and said it wasn't ideal but would be ok due to time constraints?